src/drive/styles/filelist.styl

```diff
@@ -350,6 +350,9 @@ column-width-thumbnail-bigger = 7rem
         .fil-content-body
             margin-bottom 3rem
 
+    .fil-content-body--withFabActive
+        padding-bottom calc(56px + 2rem) // Fab size + margins around
```

Comment:
Don't we have access to the fab size somewhere? Through the theme?

Reply:
non il n'y a pas de notion de size dans le theme https://next.material-ui.com/customization/default-theme/#main-content et on ne l'a pas ajouté dans cozy-ui. Et dans MUI le Fab est construit comme ça à savoir avec une valeur en dur, la taille peut être changé ce qui rajoute une classe.